### PR TITLE
feat(irs-api):[#244] Fixed chron duration to be in ISO format, added …

### PR DESCRIPTION
**Pull request by ds-psosnowski**
Changed chron to use ISO format
Adjusted commit-msg format to match github style

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - IRS can now check the readiness of external services. Use the new ``management.health.dependencies.enabled`` config entry to determine if external dependencies health checks should be checked (false by default).
   - The map of external services healthcheck endpoints can be configured with ``management.health.dependencies.urls`` property, eg. ``service_name: http://service_name_host/health``
+- Added cache mechanism for ConnectorEndpointService for fetchConnectorEndpoints method cache
+
+### Changed
+- Updated commit message format check to match github pattern
```
Comment (ds-jhartmann) [inline CHANGELOG.md:14]: That's not product related and does not need to be part of the changelog

File: charts/irs-helm/values.yaml [hunk 2]
```diff
@@ -203,7 +203,7 @@ edc:
         operator: "eq"
         rightOperand: "active"
   connectorEndpointService:
-    cacheTTL: 86400000
+    cacheTTL: PT24H # Time to live for ConnectorEndpointService for fetchConnectorEndpoints method cache
```
Comment (ds-jhartmann) [inline charts/irs-helm/values.yaml:206]: Comments need to be separated by two spaces
```suggestion
    cacheTTL: PT24H  # Time to live for ConnectorEndpointService for fetchConnectorEndpoints method cache
```

File: irs-api/src/main/resources/application.yml [hunk 3]
```diff
@@ -173,7 +173,7 @@ irs-edc-client:
         operator: "eq"
         rightOperand: "active"
   connectorEndpointService:
-    cacheTTL: 86400000
+    cacheTTL: PT24H
```
Comment (ds-jhartmann) [inline irs-api/src/main/resources/application.yml:176]: Please also add a description what this parameter does (can be the same as for values.yaml)

**Review by ds-jhartmann — CHANGES_REQUESTED** [review]

File: irs-api/src/main/resources/application.yml (hunk 3 above)
Comment (ds-jhartmann) [reply irs-api/src/main/resources/application.yml:176]: Is `@Scheduled` able to use a ISO 8601 format?

Comment (ds-psosnowski) [reply irs-api/src/main/resources/application.yml:176]: Yes it is, we use it in other places, also I've checked the documentation.
Will add comment.

**Review by ds-jhartmann — APPROVED** [review]
LGTM

**Comment by ds-jhartmann** [comment]
Chart linting is failing, please fix

File: local/development/commit-msg [hunk 4]
```diff
@@ -16,7 +16,7 @@ fi
 url="$(git remote get-url origin | sed 's/\.git/#commit-messages/')"
 
 # check semantic versioning scheme
-if ! echo "$commitTitle" | grep -qE '^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test)\([a-z0-9/-]+\)\:\[[#0-9]+\].*$'; then
+if ! echo "$commitTitle" | grep -qE '^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test)\([a-z0-9/-]+\)\:\[#[0-9]+\].*$'; then
```
Comment (ds-psosnowski) [inline local/development/commit-msg:19]: ticket number is not set to optional, any regex that I've tried that works in regex check doesn't work in bash for some reason

Comment (ds-jhartmann) [reply local/development/commit-msg:19]: This regex should work
```suggestion
if ! echo "$commitTitle" | grep -qE '^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test)\([a-z0-9/-]+\)\:(\[#[0-9]+\])?.*$'; then
```

Comment (ds-psosnowski) [reply local/development/commit-msg:19]: yea I've tested it, it works as regex but for some reason bash return it as error

Comment (ds-jhartmann) [reply local/development/commit-msg:19]: I tested this with the git hook and it is working

Comment (ds-jhartmann) [reply local/development/commit-msg:19]: Alright, lets leave it for now.

**Review by ds-jhartmann — APPROVED** [review]

**Review by ds-jhartmann — APPROVED** [review]

**Pull request merged**